### PR TITLE
Readability changes

### PR DESCRIPTION
**Pull request by Oshratn**
Changes to English and general order of the document, so we flow from user to Giant Swarm (people care about themselves).

IMO, the term 'interact with the offered services' is a little clunky and vague. What is a good one to change it to?

@othylmann does our ISO certification have anything to do with this? I am asking because the whole feel of the document is: "trust us", and I was wondering if we can add a certification to raise the level of confidence of the reader.

### CODE REVIEW

**Comment by othylmann** [comment]
@Oshratn ISO could help, yes, but it's not there yet. It would only make clear that we follow certain processes though.

**Comment by Oshratn** [comment]
Yes, I spoke with @stone-z this morning and he said that it is in the works. We'll update again once the certification is done later this year ;)

File: src/content/basics/secured-access-to-clusters/index.md [hunk 1]
```diff
@@ -35,41 +49,26 @@ VPN secured access points:
...
-### Giant Swarm API
-
-User access is provided to you over the Giant Swarm API. 
-Network access to the API endpoint is usually whitelisted to a certain range of IP addresses.
-It can also be configured to work over a VPN following the general VPN connection schema shown above under [admin access](#admin-access), where the connection to API residing in the cluster can be established only via your configured VPN.
-
-### Kubernetes API
-
-The Kubernetes API of the clusters are exposed to the customers. You have full control over the users that are created via the Giant Swarm API. Additionally, you can also manage users by connecting an external Identity Provider to the Kubernetes API.
+Access to etcd or Kubernetes API are secured based on certificates signed by Vault, to which only personnel in the Giant Swarm GitHub organization have access to.
```
Comment (stone-z) [inline src/content/basics/secured-access-to-clusters/index.md]:
```suggestion
Access to etcd or the Kubernetes API is secured based on certificates signed by Vault, to which only personnel in the Giant Swarm GitHub organization have access.
```

File: src/content/basics/secured-access-to-clusters/index.md [hunk 2]
```diff
@@ -35,41 +49,26 @@ VPN secured access points:
 
 ![](./ssh_access_process.png)  
 
-  SSH access to Control Plane allows Giant Swarm also to manage and connect to underlying Tenant Clusters of the Customer.
+SSH access to Control Plane allows Giant Swarm to manage and connect to underlying customer Tenant Clusters.
```
Comment (stone-z) [inline src/content/basics/secured-access-to-clusters/index.md]:
```suggestion
Customer Tenant Clusters are accessible only via SSH access to the Giant Swarm Control Plane. This Control Plane contains Giant Swarm's cluster management and operations platform, and controls our access to the underlying Tenant Clusters for diagnostic and "day 2" operational reasons.
```

File: src/content/basics/secured-access-to-clusters/index.md [hunk 3]
```diff
@@ -35,41 +49,26 @@ VPN secured access points:
...
 
-* **Control Plane Kubernetes API** - Usage of Kuberentes API on Control Plane also follows the authentication principles of the SSH connection.
+* **Control Plane Kubernetes API** - Usage of Kuberentes API on the Control Plane also follows the authentication principles of the SSH connection.
```
Comment (stone-z) [inline src/content/basics/secured-access-to-clusters/index.md]:
```suggestion
* **Control Plane Kubernetes API** - Usage of the Kubernetes API on the Control Plane is also secured with SSH.
```

File: src/content/basics/secured-access-to-clusters/index.md [hunk 4]
```diff
@@ -35,41 +49,26 @@ VPN secured access points:
...
 
-Cluster can be accessed by connecting to a Giant Swarm VPN server which establishes a secure connection with the jump host of the cluster.
+A cluster can be accessed by connecting to a Giant Swarm VPN server which establishes a secure connection with the jump host of the cluster.
 
 We use two different VPN providers to provide highly resilient and available support to our customers.
 
 ### Cloud provider access
 
-Currently, Giant Swarm operators - which are responsible for managing clusters lifecycle - are granted admin rights by the customer to the given cloud provider.
+Currently, Giant Swarm operators - which are responsible for managing cluster lifecycle - are granted admin rights by the customer to the given cloud provider.
```
Comment (stone-z) [inline src/content/basics/secured-access-to-clusters/index.md]:
```suggestion
Currently, Giant Swarm operators - which are responsible for managing cluster lifecycle - are granted admin rights by the customer to the given cloud provider. This is necessary to create, configure, and clean up the underlying resources (machines, networks, security groups, etc.) used by the cluster.
```

File: src/content/basics/secured-access-to-clusters/index.md [hunk 5]
```diff
@@ -1,28 +1,42 @@
...
 
 If you would like to know more about the different parts of the Giant Swarm infrastructure, please see our [operational layers article](/basics/giant-swarm-operational-layers/)
 
+## User access
+
+User access is limited to the offered APIs for interaction with your clusters. 
+
+### Giant Swarm API
+
+User access is provided to you over the Giant Swarm API.
```
Comment (stone-z) [inline src/content/basics/secured-access-to-clusters/index.md]:
```suggestion
High-level cluster management access is provided to you via the Giant Swarm API. This includes creating, scaling, and deleting your clusters, as well as other organization and user management functions.
```

File: src/content/basics/secured-access-to-clusters/index.md [hunk 6]
```diff
@@ -1,28 +1,42 @@
...
+User access is limited to the offered APIs for interaction with your clusters. 
+
+### Giant Swarm API
+
+User access is provided to you over the Giant Swarm API. 
+Network access to the API endpoint is usually whitelisted to a certain range of IP addresses. It can also be configured to work over a VPN following the general VPN connection schema shown below under [admin access](#admin-access). In this case the connection to the API residing in the cluster can be established only via your configured VPN.
+
+### Kubernetes API
+
+The Kubernetes API of the clusters are exposed to customers. You have full control over the users that are created via the Giant Swarm API. Additionally, you can also manage users by connecting an external Identity Provider to the Kubernetes API.
```
Comment (stone-z) [inline src/content/basics/secured-access-to-clusters/index.md]:
```suggestion
The Kubernetes API of each cluster is exposed to customers. You have full control over the users that are created via the Giant Swarm API. Additionally, you can also manage users by connecting an external Identity Provider to the Kubernetes API.
```

File: src/content/basics/secured-access-to-clusters/index.md [hunk 7]
```diff
@@ -1,28 +1,42 @@
...
+last-review-date = "2020-01-20"
 +++
 
 # Secure access to clusters for users and Giant Swarm support
 
-In order to provide the best service possible, Giant Swarm staff also have access to your clusters.
+It goes without saying that our customers will have secure access to their own clusters. In order to provide the best service possible, Giant Swarm staff also needs access to them.
 
-Therefore, we would like to explain this access, and what security measures are in place.
+In this document we will explain the essence of this access, and the security measures are in place to ensure adherence to the highest level of security.
```
Comment (stone-z) [inline src/content/basics/secured-access-to-clusters/index.md]:
```suggestion
In this document we will explain the nature of this access and the security measures in place to ensure management of your clusters is conducted privately and responsibly.
```

**Comment by Oshratn** [comment]
Accepted all changes from @stone-z .
Are we good to go?

**Comment by Oshratn** [comment]
@stone-z - just a last ok and I'll merge. Thanks.

**Comment by stone-z** [comment]
> @stone-z - just a last ok and I'll merge. Thanks.

It's ok with me

**Pull request merged**